myclass_second.java

```diff
@@ -0,0 +1,29 @@
+
+package test1;
+
+import java.util.Scanner;
+
+public class myclass_second {
+
+	public static void main(String[] args) {
+
+		Scanner scanner = new Scanner(System.in);
+
+		for (int i = 1; i <= 9; i++) {
+			System.out.println("-----" + i + "��-----");
+			for (int j = 1; j <= 9; j++) {
+				System.out.println(i + "x" + j + "=" + (i * j));
+			}
+		}
+		int x = scanner.nextInt();
+		int y = scanner.nextInt();
+
+		System.out.println(x + " x " + y + " = " + (x * y));
```

Comment:
이 코드를 반복문으로 돌리면 Optional1 완성입니다! 
```suggestion
		System.out.println(x + " x " + y + " = " + (x * y));
		for(int i = 1; i <= 9; i++){
 System.out.println(x + " x " + i + " = " + (x * i));
}
```